packages/nodejs/src/typeTranslation.ts

```diff
@@ -0,0 +1,282 @@
+        };
+    } else {
+        throw Error(
+            'PendingChange expected to be of type "reduce" or "remove", but found ' +
+                change.oneofKind
+        );
+    }
+}
+
+function transDelegator(
+    deleg: v2.AccountStakingInfo_Delegator
+): v1.AccountDelegationDetails {
+    return {
+        restakeEarnings: deleg.restakeEarnings,
+        stakedAmount: unwrap(deleg.stakedAmount?.value),
+        delegationTarget: transDelegatorTarget(unwrap(deleg.target)),
+        // Set the following value if deleg.pendingChange is set to true
+        ...(deleg.pendingChange && {
+            pendingChange: transPendingChange(
+                deleg.pendingChange
+            ) as v1.StakePendingChangeV1,
+        }),
+    };
+}
+
+function transAmountFraction(amount: v2.AmountFraction | undefined): number {
+    return unwrap(amount?.partsPerHundredThousand) / 100000;
+}
+
+// Todo: Smart way to do this?
+function transOpenStatus(
+    openStatus: v2.OpenStatus | undefined
+): v1.OpenStatusText {
+    switch (unwrap(openStatus)) {
+        case v2.OpenStatus.OPEN_FOR_ALL:
+            return v1.OpenStatusText.OpenForAll;
+        case v2.OpenStatus.CLOSED_FOR_NEW:
+            return v1.OpenStatusText.ClosedForNew;
+        case v2.OpenStatus.CLOSED_FOR_ALL:
+            return v1.OpenStatusText.ClosedForAll;
+    }
+}
+
+function transBaker(
+    baker: v2.AccountStakingInfo_Baker
+): v1.AccountBakerDetails {
+    const bakerInfo = baker.bakerInfo;
+    const rates = baker.poolInfo?.commissionRates;
+    const commissionRates: v1.CommissionRates = {
+        transactionCommission: transAmountFraction(rates?.transaction),
+        bakingCommission: transAmountFraction(rates?.baking),
+        finalizationCommission: transAmountFraction(rates?.finalization),
+    };
+    const bakerPoolInfo: v1.BakerPoolInfo = {
+        openStatus: transOpenStatus(baker.poolInfo?.openStatus),
+        metadataUrl: unwrap(baker.poolInfo?.url),
+        commissionRates: commissionRates,
+    };
+    return {
+        restakeEarnings: baker.restakeEarnings,
+        bakerId: unwrap(baker.bakerInfo?.bakerId?.value),
+        bakerAggregationVerifyKey: unwrapToHex(
+            bakerInfo?.aggregationKey?.value
+        ),
+        bakerElectionVerifyKey: unwrapToHex(
+            baker.bakerInfo?.electionKey?.value
+        ),
+        bakerSignatureVerifyKey: unwrapToHex(bakerInfo?.signatureKey?.value),
+        bakerPoolInfo: bakerPoolInfo,
+        stakedAmount: unwrap(baker.stakedAmount?.value),
+        // Set the following value if baker.pendingChange is set to true
+        ...(baker.pendingChange && {
+            pendingChange: transPendingChange(baker.pendingChange),
+        }),
+    };
+}
+
+export function translateAccountInfo(acc: v2.AccountInfo): v1.AccountInfo {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const accAdrRaw = Buffer.from(unwrap(acc.address?.value)) as any;
```

Comment:
Don't use type assertion here, just import 
import { Buffer } from 'buffer/';
to get rid of the error.